client/src/docs-ui/page/page.tsx

```diff
@@ -198,17 +197,19 @@ export class DocsPage {
       Object.values(routeFiltersEntry).reduce((acc, curr) => {
         return [...acc, ...curr];
       }, []);
+    console.log(currentRoute);
```

Comment:
Lets kill any `console.log(...)` that are not needed :)

Reply:
Yep, just debugging in the preview branch over here...